lib/active_model/serializer/reflection.rb

```diff
@@ -87,8 +87,8 @@ def initialize(*)
       #       meta ids: ids
       #     end
       #   end
-      def link(name, value = nil)
-        options[:links][name] = block_given? ? Proc.new : value
+      def link(name, value = nil, &block)
+        options[:links][name] = block_given? ? block : value
```

Comment:
I'm install 2.7 locally so I can figure out what you're seeing. As I wrote in the other PR, this change doesn't make sense to me

Reply:
ok, this makes more sense
> lib/active_model/serializer/reflection.rb:91: warning: Capturing the given block using Proc.new is
deprecated; use `&block` instead

Reply:
per https://github.com/ruby/ruby/commit/9f1fb0a17febc59356d58cef5e98db61a3c03550